dom-pixels.js

```diff
@@ -117,7 +117,7 @@ module.exports = function getPixels(url, type, cb) {
     cb = type
     type = ''
   }
-  var ext = path.extname(url)
+  var ext = path.extname(url.toString().split('?')[0])
```

Comment:
It's not obvious to me what `url.toString()` would do if it's not already a String.

I checked, and it looks like `path.extname(undefined)` creates an error in Node's `path` module (at least the current version), while it returns an empty string for `node-browserify` (the browserify built-in which gets included in bundles). That said, the previous line was already relying on non-standard behavior.

What about:
```js
var ext = url ? path.extname(url.split('?')[0]) : ''
```
or:
```js
var ext = typeof url === 'string' ? path.extname(url.split('?')[0]) : ''
```

🙂